src/ansys/dpf/core/operators/__init__.py

```diff
@@ -2,7 +2,6 @@
 from . import compression
 from . import filter
 from . import geo
-from . import info
```

Comment:
Hi @moe-ad it looks like you ran the operator update locally and do not have the most recent changes server-side. You'll need to run an update directly on GitHub targeting your branch.